dotcom-rendering/src/amp/components/topMeta/SeriesLink.tsx

```diff
@@ -13,22 +12,24 @@ const seriesStyle = (pillar: ArticleTheme) => css`
 	display: block;
 `;
 
-// Returns a series link if possible, and attempt to return a section link as a fallback if provided
-export const SeriesLink: React.FunctionComponent<{
+type Props = {
 	baseURL: string;
 	tags: TagType[];
 	fallbackToSection: boolean;
 	sectionLabel?: string; // required for fallback only
 	sectionUrl?: string; // required for fallback only
 	pillar: ArticleTheme;
-}> = ({
+};
+
+// Returns a series link if possible, and attempt to return a section link as a fallback if provided
+export const SeriesLink = ({
```

Comment:
If you’re adding this as new lines, maybe turn them into JSDoc as well?

Reply:
Why if we have TS? This comment I think just explains the internal logic rather than type information.